docs/source/DebuggingWithGDB.rst

```diff
@@ -185,3 +181,38 @@ breakpoints, executing commands and querying for output.
 
 When you check the output (``--show-job-log``) you can see that despite
 declaring the variable as 0, ff is injected and printed instead.
+
+
+rr wrapper
+----------
+
+It uses the ``--wrapper`` avocado functionality to run matching binaries
+inside the specified script. There is an example `rr` script in
+``/usr/share/avocado/wrappers/rr.sh`` which stores the information in
+test's `output_dir/rr` as `$bin_name.$serial_id`.
+
+Let's have a look at an example with malfunctioning qemu. Run the
+malfunctioning qemu inside avocado with wrapper::
+
+    $ avocado run "/usr/local/bin/qemu-system-aarch64 -machine virt" --wrapper /usr/share/examples/wrappers/rr.sh:*qemu*
```

Comment:
I created a card to track the need to improve `rr` support:

https://trello.com/c/3MokdHFo/795-improve-rr-support